### PR TITLE
Add greetings to README file

### PR DESCRIPTION
**Pull request by itro-dm**
this PR includes intro about me

### CODE REVIEW

**Comment by lenadgit** [comment]
Hi Dima, could you please move your greeting to the end of this file?  Also would be better to change a picture (the cat instead of a flag)

**Comment by itro-dm** [comment]
> Hi Dima, could you please move your greeting to the end of this file? Also would be better to change a picture (the cat instead of a flag)

yes sure @lenadgit , I will do it, thanks

**Comment by itro-dm** [comment]
hi @lenadgit, Ive removed the img and moved greetings to the bottom of readme

**Comment by babayK0** [comment]
@itro-dm hello, please resolve conflicts.

**Comment by itro-dm** [comment]
hello @babayK0, done

File: README.md [hunk 1]
```diff
@@ -1679,6 +1679,13 @@ Nice to meet you!
...
 
+---
+# Hi all!
+My name is [Dmitry](https://github.com/akulahere) and I'm glad to see you.
+Here it is the photo of my cat:
+![blizzard](assets/images/blizzard.png)
+
+
 ---
 #Hi there
```
Comment (babayK0) [inline README.md:1690]: One more thing, please fix typo

`#Hi there` Yours :red_circle:
#Hi there

`# Hi there` will be correct :green_circle: 
# Hi there

**Review by babayK0 — CHANGES_REQUESTED** [review]
@itro-dm

**Review by babayK0 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (itro-dm) [reply README.md]: fixed, thank you for the hint

**Pull request merged**